### PR TITLE
Adding PhilippHJS

### PR DESCRIPTION
**Pull request by mcepok**

### CODE REVIEW

File: data/players.yaml [hunk 1]
```diff
@@ -4932,3 +4932,16 @@
   twitch:
     - https://www.twitch.tv/twestaoe
   id: 403
+- name: PhilippHJS
+  country: de
+  aoeelo: 908
+  steam:
+    - 76561198014542759
```
Comment (simonsan) [inline data/players.yaml:4939]: Don't think that's necessary

File: data/players.yaml [hunk 2]
```diff
@@ -4932,3 +4932,16 @@
...
+  twitch:
+    - https://www.twitch.tv/philipphjs
+  platforms:
+    de:
+      - "209329"
+      - "1468509"
```
Comment (simonsan) [inline data/players.yaml:4945]: Question is if we add him due to not being Top200 on the primary ladder. @happyleavesaoc

The point I would make is that s*he belongs to an existing team we have already collected in `teams.json`, I would maybe even add that as an addition to the notability guidelines.

Also we would have another aliased RelicLink ID in `players.yaml`

**Review by simonsan — CHANGES_REQUESTED** [review]

File: data/players.yaml (hunk 2 above)
Comment (mcepok) [reply data/players.yaml:4945]: He is a top 200 player on https://aoe-elo.com/player/908/PhilippHJS, probably was top 200 in the 1vs1 ladder before (2k1 peak rating) and beside that, surely one of the top German players, just not very active in 1vs1 recently.

No idea what a RelicLink ID is.

File: data/players.yaml (hunk 1 above)
Comment (mcepok) [reply data/players.yaml:4939]: Just copy pasted, can remove, no problem.

File: data/players.yaml (hunk 2 above)
Comment (simonsan) [reply data/players.yaml:4943]: > No idea what a RelicLink ID is.

RelicLink is what we save under `de` in `platforms`, it's essentially the same for all Relic based multiplayer backends (AoE1DE, AoE2DE, AoE3DE, AoE4).

Comment (mcepok) [reply data/players.yaml:4943]: I see. But what's the issue? Most players have multiple IDs?

Comment (simonsan) [reply data/players.yaml:4943]: > I see. But what's the issue? Most players have multiple IDs?

Ah no issue, that was an argument to add the player. PR didn't state the reason for notability, so I only checked the RelicLink IDs for being Top200. But as you stated in a later comment, the person is Top200 on aoeelo. So everything's fine. :)

**Review by simonsan — APPROVED** [review]

**Comment by simonsan** [comment]
Thanks :)

**Pull request merged**